opennms-base-assembly/src/main/filtered/etc/report-templates/Grafana-Dashboard-Report-2ppp.jrxml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Jaspersoft Studio version 6.3.0.final using JasperReports Library version 6.3.0  -->
+<!-- 2019-07-08T15:03:09 -->
+<jasperReport xmlns="http://jasperreports.sourceforge.net/jasperreports" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://jasperreports.sourceforge.net/jasperreports http://jasperreports.sourceforge.net/xsd/jasperreport.xsd" name="Blank_A4" pageWidth="1190" pageHeight="1684" columnWidth="1150" leftMargin="20" rightMargin="20" topMargin="20" bottomMargin="20" uuid="9967c600-878d-4d17-9b00-8ca2f00e9c39">
+  <property name="com.jaspersoft.studio.unit." value="pixel"/>
+  <parameter name="reportTitle" class="java.lang.String">
+    <parameterDescription><![CDATA[Report Title]]></parameterDescription>
+    <defaultValueExpression><![CDATA["Grafana Dashboard Report"]]></defaultValueExpression>
+  </parameter>
+  <parameter name="reportDescription" class="java.lang.String">
+    <parameterDescription><![CDATA[Report Description]]></parameterDescription>
+    <defaultValueExpression><![CDATA[""]]></defaultValueExpression>
+  </parameter>
+  <parameter name="ONMS_REPORT_DIR" class="java.lang.String" isForPrompting="false">
+    <parameterDescription><![CDATA[The directory where all reports can be found]]></parameterDescription>
+    <defaultValueExpression><![CDATA["/opt/opennms/etc/report-templates/"]]></defaultValueExpression>
```

Comment:
Should be `<defaultValueExpression><![CDATA["${install.dir}/etc/report-templates"]]></defaultValueExpression>`. See https://github.com/OpenNMS/opennms/blob/34e25e095de6b72cc09e57b6c126184e40e8ee22/opennms-base-assembly/src/main/filtered/etc/report-templates/EventAnalysis.jrxml#L426.

Reply:
Fixed.